ci/integration_tests.sh

```diff
@@ -11,6 +11,11 @@ export SPEC_OPTS="--order rand --format documentation"
 export CI=true
 export OSS=true
 
+GRADLE_EXTERNAL_JDK=""
+if [ -n "$BUILD_JAVA_HOME" ]; then
+  GRADLE_EXTERNAL_JDK="-Dorg.gradle.java.home=$BUILD_JAVA_HOME"
```

Comment:
Can the existing `GRADLE_OPTS` be used here instead to avoid needing this extra ENV everywhere?

Reply:
We can avoid it, thanks to have pointed out this